index.html

```diff
@@ -10,7 +10,7 @@
       name="description"
       content="Determine the perfect build for your Dishonored 2 character before spending runes."
     />
-    <title>Dishonored 2: Power Calculator</title>
+    <title>Dishonored 2 - Power Calculator</title>
```

Comment:
💡 Purely to see if the build auto-promotes itself to the "production build" within Vercel.

Reply:
Spoiler: It auto deployed once merged to the trunk.